test/authenticated/login.js

```diff
@@ -2,17 +2,20 @@ const assert = require('assert');
 
 let test = async function(page, username, password) {
 
-    await Promise.all([
-        await page.goto('https://auth.discogs.com/login?service=https%3A//www.discogs.com/login%3Freturn_to%3D%252Fmy'),
-        page.waitFor('button.green')
-    ]);
+  await Promise.all( [
+    await page.goto( 'https://auth.discogs.com/login?service=https%3A//www.discogs.com/login%3Freturn_to%3D%252Fmy' ),
+    await page.waitFor( 'button.green' ),
+    await page.waitForSelector( '#onetrust-accept-btn-handler' )
+  ]);
 
-    await page.type('#username', username);
-    await page.type('#password', password);
-    await page.click('button.green');
+  await page.click( '#onetrust-accept-btn-handler' );
```

Comment:
Nice! Thanks for updating to deal with the OneTrust banner 👍

Reply:
I'm actually not 100% sure it's needed, but when you run it non-headless it's nice to actually see things being typed out.